.gitignore

```diff
@@ -1 +1,4 @@
 node_modules
+
+# Intellij IDEA
+.idea
```

Comment:
This is user-specific and should go into `.git/info/exclude` (local version of .gitignore) instead

Reply:
✅ done